Organize F# language guide toc/index, add F# scenarios (#26804)
dotnet/docs: docs/fsharp/using-fsharp-on-azure/deploying-and-managing.md

---
title: Deploying and Managing Azure Resources with F\#
description: Learn how to deploy and manage Azure resources with F\#
author: dsyme
ms.date: 11/04/2021
ms.custom: "devx-track-fsharp"
---

# Deploying and Managing Azure Resources with F\#

> F# may be used to configure, deploy and manage Azure resources. In this topic we give links for the F#-specific technology [Farmer](https://compositionalit.github.io/farmer/).

Reviewer: ```suggestion
F# may be used to configure, deploy, and manage Azure resources. This articles provides links for the F#-specific technology [Farmer](https://compositionalit.github.io/farmer/).
```

Other options for Azure resource deployment and management are documented in [Azure Resource Manager](/azure/azure-resource-manager/). This includes the [Bicep](/azure/azure-resource-manager/bicep) language.

## Farmer

For F# programmers, [Farmer](https://compositionalit.github.io/farmer/) is a free, open-source, community-driven technology providing an easy-to-learn library for rapidly authoring and deploying entire Azure architectures. Farmer generates [Azure Resource Manager (ARM)](/azure/azure-resource-manager) templates that define the infrastructure and configuration for your project.

> Farmer provides simple code snippets allow you to rapidly construct complex topologies, idempotent deployments, is cross-platform, and is completely backwards compatible with ARM templates. Farmer generates standard ARM templates so you can continue to use existing deployment processes.

Reviewer: ```suggestion
Farmer provides simple code snippets that allow you to rapidly construct complex topologies and idempotent deployments. Farmer is cross-platform and completely backwards compatible with ARM templates. Farmer generates standard ARM templates so you can continue to use existing deployment processes.
```
